actors/runtime/src/main/java/cloud/orbit/actors/runtime/SerializationHandler.java

```diff
@@ -144,7 +144,7 @@ public void onRead(HandlerContext ctx, final Object msg)
         catch (Throwable e)
         {
             logger.error("Error deserializing message", e);
-            logger.error(InternalUtils.hexDump(32, message.getRight(), 0, message.getRight().length));
+            logger.debug(InternalUtils.hexDump(32, message.getRight(), 0, message.getRight().length));
```

Comment:
Better to wrap this line in `isDebugEnabled` or the JVM will anyway create a hex dump from the payload and generate unnecessary garbage.

Reply:
Done